ui/app/selectors/custom-gas.js

```diff
@@ -355,3 +357,20 @@ export function getRenderableEstimateDataForSmallButtonsFromGWEI(state) {
     },
   ];
 }
+
+export function getIsEthGasPriceFetched(state) {
+  const gasState = state.gas;
+  return Boolean(
+    gasState.estimateSource === 'eth_gasprice' &&
+      gasState.basicEstimateStatus === 'READY' &&
+      getIsMainnet(state),
```

Comment:
My intent here was to make sure that the estimates fetched using eth_gasprice in this scenario are from Mainnet and not from the Testnets. Because we are displaying a warning to the user when we fetch gas price from eth_gasprice in Mainnet when Metaswaps service fails, this warning is not needed in the case of Testnets.